### PR TITLE
BZ1906407: Adding an additional resource

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.6+
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1906407
QE Ack needed: @yaoli-redhat 
Preview Link: https://deploy-preview-35112--osdocs.netlify.app/openshift-enterprise/latest/security/certificates/service-serving-certificate?utm_source=github&utm_campaign=bot_dp#add-service-certificate_service-serving-certificate

### CODE REVIEW

File: security/certificates/service-serving-certificate.adoc [hunk 1]
```diff
@@ -9,6 +9,9 @@ include::modules/customize-certificates-understanding-service-serving.adoc[level
 
 include::modules/customize-certificates-add-service-serving.adoc[leveloffset=+1]
 
+.Additional Resources
+You can use a configured service certificate to configure a secure route using reencrypt TLS termination. For more information, see xref:../../networking/routes/secured-routes.adoc#nw-ingress-creating-a-reencrypt-route-with-a-custom-certificate_secured-routes[Creating a re-encrypt route with a custom certificate].
```
Comment (bergerhoffer) [inline security/certificates/service-serving-certificate.adoc:13]: * The additional resources sections should always be a bulleted list, if you don't mind adding a bullet here
* There is both "configured" and "configure" in the first sentence. Any chance you could change either of those to a synonym?

**Comment by xingxingxia** [comment]
I'll test and review it for balancing liyao current workloads.

**Comment by sagidlow** [comment]
@xingxingxia

**Comment by xingxingxia** [comment]
Was busy in work on led subteams so didn't have time on this. Will finish test and review of this PR today.

**Comment by xingxingxia** [comment]
Did't have time to test today. Will add test when free. Adding first:
/lgtm

**Comment by xingxingxia** [comment]
Yesterday didn't have time to do the test for the doc. Today did it:
The bug title says, user can configure reencrypt route by making use of the service serving certificates. In cluster, there are already such routes, e.g.:
$ oc get route -A | grep reencrypt
openshift-console          console             console-openshift-console.apps...com                       console             https   reencrypt...

Below is normal user example:
$ cat caddyfile.conf 
:8443 {
  tls /etc/serving-cert/tls.crt /etc/serving-cert/tls.key
  root /srv/publics
  browse /test
}
:8080 {
  root /srv/public
  browse /test
}
$ oc login -u ... -p ...
$ oc new-project xxia-project
$ oc new-app quay.io/openshifttest/caddy-docker:latest
$ oc create configmap default-conf --from-file=Caddyfile=caddyfile.conf
$ oc set volumes deployment/caddy-docker --add --type configmap --configmap-name default-conf --mount-path /etc/caddy

\# annotate
$ oc annotate service caddy-docker service.beta.openshift.io/serving-cert-secret-name=server

\# make the application pod use above service serving certificate which is injected into the secret
$ oc set volumes deployment/caddy-docker --add --type secret --secret-name=server --mount-path /etc/serving-cert
$ oc get po
NAME                           READY   STATUS    RESTARTS   AGE
caddy-docker-7764dfb49-bqf7h   1/1     Running   0          26s
$ oc rsh caddy-docker-7764dfb49-bqf7h openssl s_client -connect localhost:8443 2>&1 | grep -i subject
subject=/CN=caddy-docker.xxia-project.svc

From this subject CN, we can see the application uses service serving certificate

\# create reencrypt route
$ oc create route reencrypt --service caddy-docker --port 8443
$ oc get route
NAME           HOST/PORT                                                            PATH   SERVICES       PORT   TERMINATION   WILDCARD
caddy-docker   caddy-docker-xxia-project.apps...com          caddy-docker   8443   reencrypt     None
$ openssl s_client -connect caddy-docker-xxia-project.apps...com:443 2>1 | grep -i subject
subject=/CN=*.apps...com
From this CN, we can see the route in front of the application uses ingress CA

That is, user can configur the reencrypt route by making use of the service serving certificates, as the bug's title.
So the doc LGTM. CC @yaoli-redhat fyi for above example, when you're free could have a look / study.
Thanks

**Pull request merged**